### PR TITLE
Remove ability to delete content units

### PR DESCRIPTION
**Pull request by dralley**
closes #3445
https://pulp.plan.io/issues/3445

### CODE REVIEW

File: pulpcore/pulpcore/app/viewsets/base.py [hunk 1]
```diff
@@ -258,7 +258,18 @@ class CreateDestroyReadNamedModelViewSet(mixins.CreateModelMixin,
     A customized NamedModelViewSet for models that don't support updates.
 
     A viewset that provides default `create()`, `retrieve()`, `destroy()` and `list()` actions.
+    """
+    pass
+
+
+class CreateReadNamedModelViewSet(mixins.CreateModelMixin,
```
Comment (dralley) [inline pulpcore/pulpcore/app/viewsets/base.py]: Am I the only one that finds this a liiiiittle bit ridiculous?

**Comment by daviddavis** [comment]
I tried testing this out but I am getting `AttributeError: 'NoneType' object has no attribute 'model'`.

**Comment by dralley** [comment]
@daviddavis same. no clue what is causing it though

File: pulpcore/pulpcore/app/apps.py [hunk 2]
```diff
@@ -94,11 +94,13 @@ def import_serializers(self):
     def import_viewsets(self):
         # circular import avoidance
         from pulpcore.app.viewsets import (GenericNamedModelViewSet, NamedModelViewSet,
```
Comment (dralley) [inline pulpcore/pulpcore/app/apps.py]: Man, this code smells real bad.

**Comment by dralley** [comment]
Test failure is unrelated

**Pull request merged**